cli.ts

```diff
@@ -25,8 +25,14 @@ const symbols = {
   success: kleur.green('✔'),
 };
 
+const toArray = <T extends unknown>(input: T) =>
+  Array.isArray(input) ? input : [input];
+
 sade('lighthouse-parade <url> [dataDirectory]', true)
   .version(version)
+  .example(
+    '"https://www.thebluealliance.com/team/2733/2010" --max-depth 2 --path-must-match "/team/*/2010" --path-must-match "/event/*"'
```

Comment:
I think robots are cool, but I propose we converge on a consistent pattern for URL examples. Using https://cloudfour.com is my first instinct, but I'm open to other ideas.